src/Components/Inspirations/imageApiCalls.tsx

```diff
@@ -0,0 +1,25 @@
+import './Inspirations.scss'
+
+
+const checkForError = (response: Response) => {
+  if (response.status === 404) {
+    throw (`${response.status} Error. Sorry, the image you're looking for doesn't exist.`)
+  } else if ((!response.ok && response.status >= 400) && response.status < 500) {
+    throw (`${response.status} Error. Sorry, the image you're looking for doesn't exist.`)
```

Comment:
Since the message is the same on both of these first two conditions, we could probably eliminate the first condition, since a 404 would still be covered by the second condition.

Reply:
Done! :)